Simonyi/Components/MTCS/MTDome.rst

```diff
@@ -1,6 +1,115 @@
+* *Light/Wind screen*: This component provides a rectangular clear aperture of at least 11.0 meters in the horizontal direction and at least 10.0 meters in the elevation direction. Its drive motors utilize motorized braking when decelerating under normal operations.
+* *Rear Access Doors*: These doors shall provide a clear dome access through a doorway which is capable of fully opening or fully closing in less than 90 seconds.
+* *Louvered Light Baffling Vents*: Each louver have a motorized drive system which is capable of fully opening or fully closing the louver in less than 30 seconds, under normal or marginal environmental conditions.
+* *Overhead Bridge Crane and Trolley Cane*: These are the mobile structures attached to the MTDome, they ride on a pair of horizontal support beams that run along the sides of the aperture from the MTDome shutter to the rear access door.
+* *Calibration Screen*: This is the major calibration component in the MTDome. It is a 10.4-meter diameter diffuse flat surface that is located on its interior. During operations, this screen is illuminated with both monochromatic and polychromatic light, the telescope is then pointed at the screen and images are acquired using the camera mounted on the telescope. These images can be used to determine the relative sensitivity of the telescope and camera system over the entire Simonyi Survey Telescope wavelength range.
+
+Although not part of MTDome, the Environmental Sensors Suite (ESS) keeps track of the environmental MTDome conditions. The MTDome also incorporates a high	 capacity HVAC system operated during the daytime to set the internal dome	 temperature to -1C below the expected start of night temperature.
+
+MTDome CSCs
+===========
+
+The MTDome has a dedicated Commandable SAL Component (CSC). The capacitor banks telemetry is collected via MTDome CSC. MTDomeTrajectory is the CSC in charge of enabling the MTDome following.
+
+
+MTDome Controller
+=================
+
+The Dome Control System (DCS) enable wireless communications. It is the interface to the facility lower enclosure which includes the azimuth bearing track, azimuth drive system, encoder system, slip ring and braking system. The Dome interfaces with the concrete lower enclosure through the azimuth support track, drive units, and slip rings. The MTDome CSC and GIS (Global Interlock System) communicate with the DCS (which also contains the DIS, Dome Interlock System) via ethernet. The fixed part of the DCS communicates with the rotating part of the DCS via WiFi. Currently, the MTDome Controller (DCS) is still under development.
+
+The DCS includes the following software subsystems:
+
+* AMCS - Azimuth Motion Control System which rotates the dome.
```

Comment:
Applicable to all the following components --

1. The correct format should be the name (abbreviation)
e.g. "Azimuth Motion Control System (AMCS) which rotates the dome."
"Light and Wind Screen Control System (LWSCS) which will move up and down the Light and Wind Screen once it has been installed and placed under software control." 
and so on..

2. Also "Azimuth Motion Control System (AMCS)" should be in italics as there are names of software control systems.